tests/dbal/db_tools_test.php

```diff
@@ -344,6 +344,33 @@ public function test_table_drop()
 		$this->assertFalse($this->tools->sql_table_exists('prefix_test_table'));
 	}
 
+	public function test_table_truncate()
+	{
+		$this->tools->sql_create_table('prefix_test_table',
+			array('COLUMNS' => array(
+				'foo' => array('UINT', 42)))
+		);
+
+		$this->assertTrue($this->tools->sql_table_exists('prefix_test_table'));
+
+		// insert line in table to test truncate command
+		$sql_arr = array(
+    		'foo'    => 0,   
+		);
+		$sql = 'INSERT INTO prefix_test_table' . $this->db->sql_build_array('INSERT', $sql_arr);
+		$result = $this->db->sql_query($sql);
+		$this->assertTrue($this->db->sql_fetchrow($result));
+		$this->db->sql_freeresult($result);
+
+		$this->tools->sql_table_truncate('prefix_test_table');
+
+		// table empty after truncate
+		$sql = 'SELECT * FROM prefix_test_table';
+		$result = $this->db->sql_query($sql);
+		$this->assertTrue(! $this->db->sql_fetchrow($result));
```

Comment:
`$this->assertEquals(0, $this->db->get_row_count('prefix_test_table'));` would be a bit clearer (and it also might work)